src/pages/security.md

```diff
@@ -8,6 +8,18 @@ The Apache CloudStack project understands that as a core infrastructure project,
 the application security of Apache CloudStack is of critical importance to the
 community and users.
 
+It is important to know that the project can not guarantee that it will be safe to
+
+- share access to the DataBase
+- share database dumps or other forms of backups
+- share log files
+- use any of the 3rd party integration components, such as for monitoring, storage, network and more.
+
```

Comment:
This reads a bit confusing - can you improve it to say how these integrations are further used?

Reply:
@rohityadavcloud , I updated the text. I don't think we can make it much clearer and what we can should go on the integrations page. Can you have a look at the new text, please?